examples/README.md

```diff
@@ -19,3 +19,10 @@ that uses the `package.json` in each directory to:
 [Node.js]: https://nodejs.org/en/about/
 [npm]: https://docs.npmjs.com/getting-started/what-is-npm
 [scripts]: https://docs.npmjs.com/misc/scripts
+
+## General
+
+If you `require('uswds')` in your bundled JS, you will get the minified ES5 browser bundle. (**This might be a problem if the `uswds` JS API becomes public, because it may not export.**) If you're handling ES5 conversion already or using a tool that does it automatically, you can work around this in two ways:
```

Comment:
The bold portion here can be removed, it doesn't apply to the current state of the JS.

Reply:
Done in 9f10773.